web/packages/core/src/internal/player/inner.tsx

```diff
@@ -201,7 +201,7 @@ export class InnerPlayer {
         this.debugPlayerInfo = debugPlayerInfo;
         this.onCallbackAvailable = onCallbackAvailable;
 
-        this.shadow = this.element.attachShadow({ mode: "open" });
+        this.shadow = this.element.attachShadow({ mode: "open", delegatesFocus: true });
```

Comment:
This requires Firefox 94

Reply:
That's fine, it's almost three years old and it's not like Safari where it's tied to OS version.

Reply:
The bigger concern is that this requires Safari 15 but that's probably fine too.

Reply:
Update: `delegatesFocus` from `attachShadow` has been supported since Safari 13.1, so it's alright (it's only the getter that requires 15).